packages/esm-patient-chart-app/src/visit/visit-form/visit-form.workspace.tsx

```diff
@@ -602,6 +604,14 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     ],
   );
 
+  const handleDiscard = useCallback(() => {
+    if (handleBackToSearchList) {
+      handleBackToSearchList();
```

Comment:
```suggestion
    if (openedFrom === 'service-queues-add-patient') {
    	handleBackToSearchList?.();
```

Would checking whether the visit form was [opened from the service queues app](https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-service-queues-app/src/create-queue-entry/create-queue-entry.workspace.tsx#L96) be more robust here instead?